File: CourierAmexClient/src/app/features/company/models/bank.model.ts
import { BaseEntity } from "@app/models";

export interface BankModel extends BaseEntity<number> {
  companyId?: number;
  name: string;
  phone?: string;
  address?: string;
}

export const newBank = {
  companyId:0,
  id: 0,
  name: '',
  phone: '',
  address: ''
};

File: CourierAmexAPI/CourierAmex/Models/BankModel.cs
namespace CourierAmex.Models
{
    public class BankModel: BaseEntityModel<int>
    {
        public required int CompanyId { get; set; }
        public required string Name { get; set; }
        public string? Phone { get; set; }
        public string? Address { get; set; }

    }
}
